process_under_control.py

```diff
@@ -0,0 +1,27 @@
+def check_process_control(m, data, UCL, CL, LCL, UCL_x, C_x, LCL_x):
+    # Berechne Spannweiten und Mittelwerte der Stichproben

```

Comment:
Man könnte noch einen Docstring einfügen um die Funktion zu beschreiben. Dazu werden in Python jeweils `"""` am Anfang und Ende eingefügt. Es gibt auch verschiedenste Guidelines. Typisch ist das von Google ([Beispiel Google](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) und Numpy ([Style Guide Numpy](https://numpydoc.readthedocs.io/en/latest/format.html)). Hier könnte das so aussehen:

```
""" Kontrolliert ob die Daten innerhalb der 3-sigma Kontrollgrenzen liegen und schreibt eine Nachricht in die Console.
"""
```